azure-iot-device/azure/iot/device/common/pipeline/pipeline_stages_base.py

```diff
@@ -17,7 +17,7 @@
 from . import pipeline_ops_base, pipeline_ops_mqtt
 from . import pipeline_thread
 from . import pipeline_exceptions
-from azure.iot.device.common import handle_exceptions
+from azure.iot.device.common import handle_exceptions, transport_exceptions
```

Comment:
I'm not concerned about this right now, but I think at some point we should revisit some of how and where these exceptions are defined - I'm a bit worried about certain things creeping beyond the scope of where they should be used.